Sources/ScipioKit/Producer/FrameworkProducer.swift

```diff
@@ -303,6 +339,13 @@ struct FrameworkProducer {
         return []
     }
 
+    private func cacheFrameworksIfNeeded(_ targets: Set<CacheSystem.CacheTarget>, cacheSystem: CacheSystem) async {
+        let storagesWithProducer = cachePolicies.storages(for: .producer)
+        if !storagesWithProducer.isEmpty {
+            await cacheSystem.cacheFrameworks(targets, to: storagesWithProducer)
+        }
```

Comment:
[Q] I'm concerned that the way storages are handled is different for consumers and producers.

For consumers, storages are evaluated in order and the top ones are used.
On the other hand, producers try to write to all declared storages at the same time.

Is this difference in behavior intentional?

Reply:
It's intentional. If we don't cache built frameworks to 2nd storages and later, the storage will not be able to restore the frameworks so saving to all storages will be required I think.